internal/protogen/exporter.go

```diff
@@ -295,28 +295,36 @@ func (x *sheetExporter) exportField(depth int, tagid int, field *tableaupb.Field
 			x.Imports[typeInfo.ParentFilename] = true
 		}
 	} else {
-		if field.Fields != nil {
-			// iff field is a map or list and message type is not imported.
-			nestedMsgName := prefix + "." + typeName
+		// iff field is a map or list and message type is not imported.
```

Comment:
Move this comment line to below of `case field.Fields != nil:`

Reply:
done